_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/pages/_katacoda.scss

```diff
@@ -0,0 +1,121 @@
+/* Katacoda */
+.katacoda-courses {
```

Comment:
There is a change in background on this page where .katacoda-courses is in used: https://developer-preview-3455.ext.us-west.dc.preprod.paas.redhat.com/courses/service-mesh/

Reply:
@staceymosier I have pushed a fix for the mentioned issue, please review.